README.md

```diff
@@ -16,6 +18,20 @@ gene_names = data.gene_names_at_locus(contig=6, position=29945884)
 exon_ids  = data.exon_ids_of_gene_name('HLA-A')
 ```
 
+# Installation
+
+You can install PyEnsembl using [pip](https://pip.pypa.io/en/latest/quickstart.html):
+
+```sh
+pip install pyensembl
```

Comment:
I'm not confident this will properly install all the requirements; I've had issues with a number of the libraries having incomplete `setup.py`s. Trying to test that out now, though having issues with `python setup.py develop --uninstall`ing